resources/dw/schema_hive_vw.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 Cloudera. All Rights Reserved.
+//
+// This file is licensed under the Apache License Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, either express or implied. Refer to the License for the specific
+// permissions and limitations governing your use of the file.
+
+package dw
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+)
+
+var hiveSchema = schema.Schema{
```

Comment:
@gregito why do we specify a `var` to define the schema? 
Is there a convention to be so explicit instead of inlining this here -> https://github.com/cloudera/terraform-provider-cdp/pull/135/files#diff-db352c5d0628c8f77a3f44b0e3adaeabc45e6f2034c5a7d7adf25d04792f05b0R46

Reply:
I would be happy with inlining! TBH these many files give me a bit of a headache because it is difficult to find the pieces that belong together. In the terraform tutorial project these were all kept in one file.

Reply:
we can leave it as is, this is going to be a pretty long schema